omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -54,7 +55,7 @@
 	// -------------------------------------------------------------------------
 	// REST implementation
 	// -------------------------------------------------------------------------
-	@RequestMapping(value = "/rest/owa/applist", method = RequestMethod.GET)
+    @RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/owa/applist", method = RequestMethod.GET)
```

Comment:
Just create a new controller and leave this one un touched because we want to preserve existing functionality in order not to break clients that may already be using it.

Reply:
Okay @dkayiwa. I think implementation remains the same.